junit-jupiter-params/src/main/java/org/junit/jupiter/params/converter/LocaleConversionFormat.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2015-2023 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.params.converter;
+
+import static org.apiguardian.api.API.Status.INTERNAL;
+
+import org.apiguardian.api.API;
+
+/**
+ * Enumeration of {@link java.util.Locale} conversion formats.
+ *
+ * @since 5.11
+ */
+@API(status = INTERNAL, since = "5.11")
```

Comment:
Is it correct to mark this as an internal API as the users are supposed to set these values in the config property only and not rely on the enum?